training/mock.netsvc/ivy.xml

```diff
@@ -0,0 +1,19 @@
+<ivy-module version="2.0" >
+  <info organisation="com.servicemesh" module="${ant.project.name}" />
+  <dependencies>
+
+	<dependency org="log4j" name="log4j" rev="1.2.17" />
+    <dependency org="org.eclipse" name="org.eclipse.osgi" rev="3.8.0.v20120529-1548">
+       <artifact name="org.eclipse.osgi" type="jar" ext="jar" />
+    </dependency>		
+
+
+	<dependency org="com.servicemesh" name="core" rev="10.2.2.+" force="true" >
+		<artifact name="core" type="jar" ext="jar" />
+	</dependency>		
```

Comment:
at some point, we should figure out how to get this as a  transitive dependency rather than specifying it here.